core/time/time.odin

```diff
@@ -360,7 +356,13 @@ _abs_date :: proc "contextless" (abs: u64, full: bool) -> (year: int, month: Mon
 	return
 }
 
+is_leap_year :: proc "contextless" (year: int) -> bool {
+	return !((year & 3 > 0) || (year & 5 > 0) && !(year % 25 == 0))
+}
```

Comment:
Why did you remove the clearer logic and replace it with "magic" (I understand what it does)?

Reply:
An understanding that partially bitwise arithmetic would be faster than completely modular arithmetic.